warrior/Framework/ClassUtils/WNetwork/warrior_cli_class.py

```diff
@@ -1606,8 +1606,13 @@ def send_command(self, command, start_prompt, end_prompt,
             except Exception as exception:
                 print_exception(exception)
             else:
-                response = self.target_host.before
-                response = str(response) + str(self.target_host.after)
+                response = str(self.target_host.before)
+                # When the command gets timed out, the pexpect.TIMEOUT exception
+                # will be raised and it is set to the after property of spawn object
+                if self.target_host.after == self.pexpect.TIMEOUT:
+                    pNote("EXCEPTION !! Command Timed Out", 'error')
+                else:
+                    response = str(response) + str(self.target_host.after)
```

Comment:
no need to convert response to string here.

Reply:
Done.